bin/collator/src/parachain/service.rs

```diff
@@ -290,14 +289,13 @@ async fn build_relay_chain_interface(
 /// This is the actual implementation that is abstract over the executor and the runtime api.
 #[cfg(not(feature = "evm-tracing"))]
 #[sc_tracing::logging::prefix_logs_with("Parachain")]
-async fn start_node_impl<RuntimeApi, Executor, BIQ, BIC>(
+async fn start_node_impl<RuntimeApi, Executor, BIQ>(
```

Comment:
How come you didn't adopt the same approach with `evm-tracing` feature flag as in the local-chain service code, reducing code duplication even more?

Reply:
because of [this](https://github.com/AstarNetwork/Astar/pull/1263/files#diff-e7ed72ccf74167731e3cfe0770979224242dfd278d03768448761ad634cac229R678-R679) I can have where and cfg feature together. It's still experimental feature

Reply:
Ah, I see, so the same old reason 🙈